editor/src/core/shared/array-utils.ts

```diff
@@ -36,7 +36,7 @@ export function flatMapArray<T, U>(
 
 export function mapDropNulls<T, U>(
```

Comment:
I really think we need to create a `filterMap` (or `mapFilter`), because that's one of the main uses of this function

Reply:
I actually like this API because the callback function can just return null to filter something out.

Reply:
Right, which was the point of this function in the first place. I mean that since we use it heavily for mapping an array and filtering it at the same time, we should make a function (or pair of functions) more clearly designed for that.